projects/driver_controls/inc/can_fsm.h

```diff
@@ -3,5 +3,22 @@
 // FSM for transmitting driver input data over CAN
```

Comment:
Do you mind renaming `can_fsm`? It sounds like it belongs to the base CAN driver implementation. My current branch already has a can_fsm module.